samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,73 @@
+.. _nrf52-power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+This sample demonstrares a power manager app that uses the Zephyr
+power management infrastructure to enter into Low Power state.
+
+This app will cycle through following power schemes each time idle thread calls
+_sys_soc_suspend() hook function :-
+
+1. Low Power State:-
+   Low Power State is SOC specific state, In state is transparent to devices.
+   SOC and devices do not lose context in this Mode. SOC supports following
+   two Low Power states :-
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
```

Comment:
needs a blank line before this line that's starting a list, also, remove the ``-`` at the end of the previous line